### ISSUE TITLE
Using ABSStore without prefix

### ISSUE THREAD

**Issue by RPrudden** [zarr-developers/zarr-python#527]
I'm trying to use ABSStore to load a zarr database stored in the top level of a blob store. At the moment this doesn't seem to be possible - even if I use an empty prefix, the listdir method will add a '/' judging from the code [here](https://zarr.readthedocs.io/en/stable/_modules/zarr/storage.html#ABSStore).

```    
def listdir(self, path=None):
        store_path = normalize_storage_path(path)
        # prefix is normalized to not have a trailing slash
        dir_path = self.prefix
        if store_path:
            dir_path = dir_path + '/' + store_path
        dir_path += '/'
        items = list()
        for blob in self.client.list_blobs(self.container, prefix=dir_path, delimiter='/'):
            if '/' in blob.name[len(dir_path):]:
                items.append(self._strip_prefix_from_path(
                    blob.name[:blob.name.find('/', len(dir_path))], dir_path))
            else:
                items.append(self._strip_prefix_from_path(blob.name, dir_path))
        return items```

I guess the assumption is that the data will always be stored in a subdirectory within the blob store. Would it be possible to support storing in the top level of a blob store?

**Comment by shikharsg** [comment]
Hi @RPrudden , good timing for reporting this. This was raised just yesterday issue #525  and fixed and merged into master just today PR #526 . It will make it into the next release. Meanwhile you can install from the master branch on github, if that is okay with you.

**Comment by RPrudden** [comment]
Great, thanks!
